apps/examples/mpu/mpu_test.c

```diff
@@ -22,36 +22,43 @@
 #include <stdio.h>
 #include <unistd.h>
 #include <semaphore.h>
-
-#if !defined(CONFIG_MPUTEST_KERNEL_CODE_ADDR) || !defined(CONFIG_MPUTEST_KERNEL_DATA_ADDR) ||\
-	(!defined(CONFIG_MPUTEST_APP_ADDR) && defined(CONFIG_APP_BINARY_SEPARATION))
-#error "Address not defined for MPU TEST"
-#endif
+#include <fcntl.h>
+#include <tinyara/testcase_drv.h>
+#include <tinyara/mpu_test.h>
 
 #ifdef CONFIG_BUILD_KERNEL
 int main(int argc, FAR char *argv[])
 #else
 int mpu_tc_main(int argc, char *argv[])
 #endif
 {
-	volatile uint32_t *address;
 	char ch;
+	int tc_fd = open(KERNEL_TC_DRVPATH, O_WRONLY);
```

Comment:
Taejun is right... There is no open in kernel_test_drv.
But this is not related to this commit. We should check/modify with another commit.

Anyway, originally `KERNEL_TC_DRVPATH` is for kernel tc. I think we'd better change the path name if you want to use in here, like `TESTCASE_DRVPATH` or something.

Reply:
We checked that current driver is okay. (I mean we can use open without drv implementation.)
Please check my second comment only.

Reply:
Done.